src/core/time.d

```diff
@@ -1506,10 +1506,7 @@ public:
            units == "hnsecs" ||
            units == "nsecs")
     {
-        static if (units == "nsecs")
-            return convert!("hnsecs", "nsecs")(_hnsecs);
-        else
-            return getUnitsFromHNSecs!units(_hnsecs);
+        return convert!("hnsecs", units)(_hnsecs);
```

Comment:
The body of `getUnitsFromHNSecs` was
https://github.com/dlang/druntime/blob/525b89e4fce7133479fa73446a19f0db5d9b99bc/src/core/time.d#L3733-L3744